network/packet_proxy.go

```diff
@@ -53,13 +53,13 @@ type PacketRequestSender interface {
 	Close() error
 }
 
-// PacketResponseReceiver receives UDP response packets from the [PacketProxy]. It is usually implemented by the
-// upstream component (such as a network stack). When creating a new UDP session, a valid instance
-// PacketResponseReceiver is passed to the PacketProxy.NewSession function. Then the [PacketProxy] must use this
-// instance to send UDP responses all the time; and it must call the Close function to indicate there will be no more
-// responses send to this receiver.
+// PacketResponseReceiver receives UDP response packets from the [PacketProxy]. It is usually implemented by an
+// upstream component (such as a network stack). When a new UDP session is created, a valid instance of
+// PacketResponseReceiver is passed to the PacketProxy.NewSession function. The [PacketProxy] must then use this
+// instance to send UDP responses. It must then call the Close function to indicate that there will be no more
+// responses sent to this receiver.
 //
-// Multiple goroutines may invoke methods on a PacketResponseReceiver simultaneously.
+// Multiple goroutines can simultaneously invoke methods on a PacketResponseReceiver.
```

Comment:
[PacketResponseReceiver]

Reply:
Type names should not be wrapped in `[]` in the documentation that describes the type itself.